uber/site_sections/registration.py

```diff
@@ -318,12 +319,18 @@ def check_in(self, session, id, age_group, group, badge_num=None, message=''):
                     message = 'That badge number already belongs to ' + maybe_dupe.first().full_name
 
             if group:
-                session.match_to_group(attendee, session.group(group))
+                g = sesson.group(group)
+                if g.amount_unpaid:
+                    message = 'That group has an outstanding balance of ${}'.format(g.amount_unpaid)
+                else:
+                    session.match_to_group(attendee, g)
```

Comment:
We should probably go through all of the groups who aren't paid up and send them reminder emails or something, and then eliminate their unpaid badges before the event if they don't pay.

This is one area where we don't have an automated email already, for a couple of reasons.  Mainly, the issue is that each automated email only gets sent once, and it's possible for a group to have multiple occasions where they have an outstanding balance.  But sending a single reminder email seems better than sending no reminder emails.